apiserver/src/main/java/org/dependencytrack/persistence/DefaultObjectGenerator.java

```diff
@@ -362,4 +368,13 @@ private void loadDefaultNotificationPublishers(final QueryManager qm) {
             LOGGER.error("An error occurred while synchronizing a default notification publisher", e);
         }
     }
+
+    /**
+     * Checks if metrics partitions exist for today and create if they don't exist.
+     */
+    void ensureMetricsPartitions() {
+        useJdbiHandle(handle -> handle.attach(MetricsDao.class).createMetricsPartitionsForDate(
+                LocalDate.now().toString(),
+                LocalDate.now().plusDays(1).toString()));
```

Comment:
Can we create partitions for *tomorrow* here as well, analogue to `MetricsMaintenanceTask`?